file_manager/launched_files.py

```diff
@@ -0,0 +1,35 @@
+from .exceptions import FileAlreadyRunningException, FileNotRunningException
+
+
+class SingletonMeta(type):
+
+    _instances = {}
+
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            instance = super().__call__(*args, **kwargs)
+            cls._instances[cls] = instance
+        return cls._instances[cls]
```

Comment:
You have instance of class in variable "instance". Why we search instance in dict again?